lib/constants/constants.dart

```diff
@@ -103,6 +105,97 @@ class WidgetTypes {
   static const String TEXT_WIDGET = "TextWidget";
   static const String WORLD_CLOCK_WIDGET = "WorldClockWidget";
   static const String PERSON_DRAW_WIDGET = "PersonDrawWidget";
+  static const String JIRA_BUCKETS_WIDGET = "JiraBucketsWidget";
+  static const String ZABBIX_WIDGET = "ZabbixWidget";
+  static const String IFRAME_EMBED_WIDGET = "IframeEmbedWidget";
+  static const String RANDOM_PICKER_WIDGET = "RandomPickerWidget";
```

Comment:
In my opinion, this whole section is redundant and should be deleted because a list of available widgets is coming from backend:
`/config.json` -> `widgets.widgetTypes` property